examples/psci/BasicEval.purs

```diff
@@ -0,0 +1,7 @@
+import Prelude
+import Data.Array
+
+let fac n = foldl mul 1 (1..n) in fac 10 -- 3628800
+
+fac n = foldl mul 1 (1..n)
+fac 10 -- 3628800
```

Comment:
Separated line comments may be safer to note the eval tests! I will update this PR.